docs/conceptual/runtime/contracts/contracts_module.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Contracts Module
+---
+
+The [SRML Contracts module](https://substrate.dev/rustdocs/master/srml_contracts/index.html)
```

Comment:
Shouldn't this and all other occurrences of `SRML` be `PAINT` now? From what I understood the re-name is final for now...

Reply:
What about the proposed name PALETTE?

Reply:
Good question...
Ok, rephrased: As of today SRML is not present in the substrate codebase anymore (apart from not updated code comments), so it probably makes sense to update the 2.0 docs to the new name, whatever that may be.

Reply:
Separate PR can do this once we know a final name.